android/sdk/src/main/cpp/app/organicmaps/sdk/Framework.cpp

```diff
@@ -1797,6 +1797,16 @@ JNIEXPORT jint Java_app_organicmaps_sdk_Framework_nativeGetBookmarksTextPlacemen
 {
   return std::to_underlying(Framework::GetBookmarksTextPlacement());
 }
+
+JNIEXPORT void Java_app_organicmaps_sdk_Framework_nativeSetShowDownloadedRegions(JNIEnv *, jclass, jboolean show)
+{
+  frm()->SetShowDownloadedRegionsEnabled(static_cast<bool>(show));
+}
+
+JNIEXPORT jboolean Java_app_organicmaps_sdk_Framework_nativeGetShowDownloadedRegions(JNIEnv *, jclass)
+{
+  return static_cast<jboolean>(frm()->ShowDownloadedRegions());
+}
```

Comment:
We have 4 naming variants here describing one thing :)
```
SetXXX, SetXXXEnabled, GetXXX, XXX
```

I'd prefer to have IsXXX, SetXXX: ```IsShowDownloadedRegions, SetShowDownloadedRegions``` everywhere
.

Reply:
Done, PTAL